src/test/java/app/bpartners/api/unit/wms/GeoserverImageSourceTest.java

```diff
@@ -0,0 +1,123 @@
+package app.bpartners.api.unit.wms;
+
+import static app.bpartners.api.endpoint.rest.model.AreaPictureImageSource.GEOSERVER;
+import static app.bpartners.api.endpoint.rest.model.AreaPictureImageSource.GEOSERVER_IGN;
+import static app.bpartners.api.endpoint.rest.model.ZoomLevel.HOUSES_0;
+import static app.bpartners.api.model.exception.ApiException.ExceptionType.SERVER_EXCEPTION;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import app.bpartners.api.endpoint.rest.model.GeoPosition;
+import app.bpartners.api.file.FileDownloaderImpl;
+import app.bpartners.api.model.AreaPicture;
+import app.bpartners.api.model.AreaPictureMapLayer;
+import app.bpartners.api.model.exception.ApiException;
+import app.bpartners.api.service.WMS.ArcgisZoom;
+import app.bpartners.api.service.WMS.Tile;
+import app.bpartners.api.service.WMS.imageSource.GeoserverImageSource;
+import app.bpartners.api.service.converter.XYZToBoundingBox;
+import java.io.File;
+import java.math.BigDecimal;
+import java.net.URI;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.core.io.FileSystemResource;
+
+class GeoserverImageSourceTest {
```

Comment:
This is an IT because you are mocking beans and use `@MockBean` instead of `@Mock`